src/main/java/engine/FormplayerConfigEngine.java

```diff
@@ -57,26 +36,13 @@ public FormplayerConfigEngine(IStorageIndexedFactory storageFactory,
         ReferenceManager.instance().addReferenceFactory(formplayerArchiveFileRoot);
     }
 
-    private void recordDownloadBreadcrumbs(String downloadUrl, String responseCode) {
-        Map<String, String> data = new HashMap<String, String>();
-        data.put("downloadUrl", downloadUrl);
-        data.put("responseCode", responseCode);
-
-        BreadcrumbBuilder builder = new BreadcrumbBuilder();
-        builder.setData(data);
-        builder.setCategory("application-install");
-        builder.setMessage("Downloading application from URL " + downloadUrl);
-        SentryUtils.recordBreadcrumb(raven, builder.build());
-    }
-
     @Override
     protected String downloadToTemp(String resource) {
         try {
             URL url = new URL(resource);
             HttpURLConnection conn = (HttpURLConnection) url.openConnection();
             conn.setInstanceFollowRedirects(true);  //you still need to handle redirect manully.
             HttpURLConnection.setFollowRedirects(true);
-            recordDownloadBreadcrumbs(resource, conn.getResponseMessage());
```

Comment:
was having trouble auto wiring this class for some reason. even after including it in the component scan. don't think it super important though